format/puffin-spec.md

```diff
@@ -121,7 +121,9 @@ distinct values converted to bytes using Iceberg's single-value serialization.
 
 The blob metadata for this blob may include following properties:
 
-- `ndv`: estimate of number of distinct values, derived from the sketch.
+- `ndv`: estimate of number of distinct values, derived from the sketch,
```

Comment:
Perhaps providing examples of allowed and not allowed values would also help with the clarification

Reply:
Or maybe a grammar?